<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(67, 67, 67);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_login">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>LOGIN</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>241</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_user_login">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="placeholderText">
           <string>username</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_pass_login">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>password</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_login">
        <property name="minimumSize">
         <size>
          <width>139</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(3, 218, 197);
color: rgb(15, 15, 15);</string>
        </property>
        <property name="text">
         <string>login</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSignUp">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>SIGN UP</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>261</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_user_signup">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="placeholderText">
         <string>username</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_pass_signup">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_confirm_pass_signup">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>confirm password</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_signup">
        <property name="minimumSize">
         <size>
          <width>139</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>249</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(3, 218, 197);
color: rgb(15, 15, 15);</string>
        </property>
        <property name="text">
         <string>sign up</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>651</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>135</y>
       <width>231</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_SIGNUP">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(3, 218, 197);
color: rgb(15, 15, 15);</string>
        </property>
        <property name="text">
         <string>SIGN UP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_LOGIN">
        <property name="minimumSize">
         <size>
          <width>229</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>229</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(3, 218, 197);
color: rgb(15, 15, 15);</string>
        </property>
        <property name="text">
         <string>LOGIN</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>631</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(64, 64, 64);
font: 75 14pt &quot;Arial&quot;;
background-color: rgb(187, 134, 252);


</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>10</number>
     </property>
     <property name="text">
      <string>If you have already sign up, click the (login) to log in.
Otherwise, to entering complete your sign up by click (sign up) .</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_LOGIN</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_login</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_LOGIN</sender>
   <signal>clicked()</signal>
   <receiver>groupBoxSignUp</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_SIGNUP</sender>
   <signal>clicked()</signal>
   <receiver>groupBoxSignUp</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_SIGNUP</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_login</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
